sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -197,6 +200,10 @@ public synchronized void stop() {
           LOG.error("Error stopping Http server: ", e);
         }
       }
+      if (serverThread != null) {
+        serverThread.interrupt();
+        serverThread = null;
+      }
```

Comment:
Sorry, but this is inconsistent with HIVE-19776 because it invokes `stopServer` after this `interrupt()`.

I know that you moved this at the last commit, but I'm wondering if we can have this before `httpServer.stop()` to be consistent with HIVE-19776.

Reply:
thanks @dongjoon-hyun. I will check this soon after Chinese New Year Festival😁